config/src/main/kotlin/org/springframework/security/config/annotation/web/HttpSecurityDsl.kt

```diff
@@ -76,6 +78,31 @@ class HttpSecurityDsl(private val http: HttpSecurity, private val init: HttpSecu
 
     var authenticationManager: AuthenticationManager? = null
 
+    /**
+     * Applies a [SecurityConfigurerAdapter] to this [HttpSecurity]
+     *
+     * Example:
+     *
+     * ```
+     * @Configuration
+     * @EnableWebSecurity
+     * class SecurityConfig : WebSecurityConfigurerAdapter() {
```

Comment:
The DSL has recently been updated to no longer include examples with `WebSecurityConfigurerAdapter` (see gh-11646).

Reply:
aha, just removed WebSecurityConfigurerAdapter from the docs.